recipes_source/xeon_run_cpu.rst

```diff
@@ -0,0 +1,302 @@
+
+.. code-block:: console
+
+   $ conda install conda-forge::jemalloc
+
+Quick Start Example Commands
+----------------------------
+
+1. To run single-instance inference with 1 thread on 1 CPU core (only Core #0 would be used):
+
+.. code-block:: console
+
+   $ python -m torch.backends.xeon.run_cpu --ninstances 1 --ncores-per-instance 1 <program.py> [program_args]
+
+2. To run single-instance inference on a single CPU node (NUMA socket):
+
+.. code-block:: console
+
+   $ python -m torch.backends.xeon.run_cpu --node-id 0 <program.py> [program_args]
+
+3. To run multi-instance inference, 8 instances with 14 cores per instance on a 112-core CPU:
+
+.. code-block:: console
+
+   $ python -m torch.backends.xeon.run_cpu --ninstances 8 --ncores-per-instance 14 <program.py> [program_args]
+
+4. To run inference in throughput mode, in which all the cores in each CPU node set up an instance:
+
+.. code-block:: console
+
+   $ python -m torch.backends.xeon.run_cpu --throughput-mode <program.py> [program_args]
+
+.. note::
+
+   Term "instance" here doesn't refer to a cloud instance. This script is executed as a single process which invokes multiple "instances" which are formed from multiple threads. "Instance" is kind of group of threads in this context.
+
+Using ``torch.backends.xeon.run_cpu``
+-------------------------------------
+
+The argument list and usage guidance can be shown with the following command:
+
+.. code-block:: console
+
+   $ python -m torch.backends.xeon.run_cpu –h
+   usage: run_cpu.py [-h] [--multi-instance] [-m] [--no-python] [--enable-tcmalloc] [--enable-jemalloc] [--use-default-allocator] [--disable-iomp] [--ncores-per-instance] [--ninstances] [--skip-cross-node-cores] [--rank] [--latency-mode] [--throughput-mode] [--node-id] [--use-logical-core] [--disable-numactl] [--disable-taskset] [--core-list] [--log-path] [--log-file-prefix] <program> [program_args]
+
+The command above has the following positional arguments:
+
++------------------+---------------------------------------------------------+
+| knob             | help                                                    |
++==================+=========================================================+
+| ``program``      | The full path of the program/script to be launched.     |
++------------------+---------------------------------------------------------+
+| ``program_args`` | All the arguments for the program/script to be launched.|
++------------------+---------------------------------------------------------+
+
+Explanation of the options
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The generic option settings (knobs) include the following:
+
++----------------------+------+---------------+-------------------------------------------------------------------------------------------------------------------------+
+| knob                 | type | default value | help                                                                                                                    |
```

Comment:
not sure why by the header row (Defaul value) in this table looks a bit weird:

<img width="874" alt="Screenshot 2024-06-20 at 10 31 03 AM" src="https://github.com/pytorch/tutorials/assets/5317992/520d3f5e-46e7-471f-8260-e95f3334718f">

I myself prefer [list-tables](https://sublime-and-sphinx-guide.readthedocs.io/en/latest/tables.html#list-table-directive) for tables like this, For example:

```
.. list-table::
   :widths: 25 10 15 50
   :header-rows: 1
   * - knob
     - type
     - default value
     - help
   * - ``-h``, ``--help``
     - 
     - 
     - Show the help message and exit.
   * - ``-m``, ``--module``
     - 
     - 
     - Changes each process to interpret the launch script as a python module, executing with the same behavior as "python -m".
   * - ``--no-python``
     - bool
     - False
     - Do not prepend the program with "python" - just exec it directly. Useful when the script is not a Python script.
   * - ``--log-path``
     - str
     - ''
     - The log file directory. Default path is ``''``, which means disable logging to files.
   * - ``--log-file-prefix``
     - str
     - 'run'
     - log file name prefix.
 ```

Reply:
Thanks. I pushed a commit updating the formatting syntax of the tables per your advice.
Not deleting the old formatted tables for comparison.